### PR TITLE
Enable radar reflectivity output

### LINKED ISSUE #1513: Missing simulated radar reflectivity
**Expected behavior**
Output files should have valid simulated reflectivity (`REFC` and `REFD`), or it should not be a grib record.

**Current behavior**
The master file contains undefined fields for REFC and REFD.

**Machines affected**
All

**To Reproduce**
Run any experiment in workflow.

**Context**
The undefined variables were first identified by @TerrenceMcGuinness-NOAA during CI testing build-out. Additional undefined variables he found are resolved by #1509.

**Detailed Description**
```
Orion-03-09 master/>wgrib2 -stats -var gfs.t00z.master.grb2f024 | grep -P 'undef=73728'
7:194243:ndata=73728:undef=73728:mean=0:min=0:max=0:REFD
8:203638:ndata=73728:undef=73728:mean=0:min=0:max=0:REFD
9:213033:ndata=73728:undef=73728:mean=0:min=0:max=0:REFC
822:49446144:ndata=73728:undef=73728:mean=0:min=0:max=0:REFD
823:49455539:ndata=73728:undef=73728:mean=0:min=0:max=0:REFD
```

All master files (and grib files derived from it) have the same missing fields.

**Possible Implementation**
Either add the needed fields or, if GFS is not intended to produce simulated reflectivity, stop producing grib records for it.

### PR DESCRIPTION
**Pull request by WalterKolczynski-NOAA**
# Description
Turns on radar reflectivity output from FV3. This eliminates missing values appearing in the grib2 files.

Resolves #1513

# Type of change
- New feature (adds functionality)

# Change characteristics
- Is this a breaking change (a change in existing functionality)? NO
- Does this change require a documentation update? NO

# How has this been tested?
- Forecast-only on Orion

# Checklist
- [x] Any dependent changes have been merged and published
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] New and existing tests pass with my changes
- [x] I have made corresponding changes to the documentation if necessary

### CODE REVIEW

**Comment by WalterKolczynski-NOAA** [comment]
grib2 files now have valid data for the associated fields:
```
(gfs_workflow) Orion-login-2 master/>wgrib2 -match ':REF' -stats -var gfs.t12z.master.grb2f024 
7:67551:ndata=18432:undef=0:mean=-16.6312:min=-20:max=35.09:cos_wt_mean=-17.0096:REFD
8:83547:ndata=18432:undef=0:mean=-16.6319:min=-20:max=38.18:cos_wt_mean=-17.0171:REFD
9:99494:ndata=18432:undef=0:mean=-14.5027:min=-20:max=41.66:cos_wt_mean=-14.8828:REFC
822:14312036:ndata=18432:undef=0:mean=-18.8605:min=-20:max=27.96:cos_wt_mean=-18.6435:REFD
823:14319277:ndata=18432:undef=0:mean=-17.3793:min=-20:max=31.62:cos_wt_mean=-17.6857:REFD
```

**Review by RuiyuSun — APPROVED** [review]
The changes look fine to me.

**Review by WenMeng-NOAA — APPROVED** [review]

File: ush/parsing_namelists_FV3.sh [hunk 1]
```diff
@@ -251,7 +251,7 @@ EOF
   cat >> input.nml << EOF
   iovr         = ${iovr:-"3"}
   ltaerosol    = ${ltaerosol:-".false."}
-  lradar       = ${lradar:-".false."}
+  lradar       = ${lradar:-".true."}
```
Comment (aerorahul) [inline ush/parsing_namelists_FV3.sh:254]: this is no longer optional with the update in `diag_table`, correct?
In other words, if `lradar=.false.` and `diag_table` has `refl_10cm`, will the model still run successfully?

Comment (RuiyuSun) [reply ush/parsing_namelists_FV3.sh:254]: lradar=.true. is needed to have correct refl_10cm  values. Can this be made optional? When lradar = .true., refl_10cm will be added to the diag_table?

Comment (WalterKolczynski-NOAA) [reply ush/parsing_namelists_FV3.sh:254]: When I accidentally left this as false during testing, the model still completed successfully (and the fields were missing from the output). So both settings are valid.

Comment (aerorahul) [reply ush/parsing_namelists_FV3.sh:254]: Interesting. That means, if `lradar=.false.`, the `refl_10cm` values are incorrect even if the model runs successfully.

**Review by aerorahul — APPROVED** [review]
It works.

**Pull request merged**